lib/jira.ts

```diff
@@ -181,4 +181,8 @@ export class Jira {
         }
         return true;
     }
+
+    public static getFinalStatuses(config: Config): string[] {
```

Comment:
Actually, this gets final status names instead of final statuses objects. So I would rename this function as getFinalStatusNames.